charts/lustre-csi-driver/templates/_helpers.tpl

```diff
@@ -0,0 +1,62 @@
+{{/*
```

Comment:
It doesn't look like you're using any of these macros.

Reply:
Can I just delete this file then? `helm create` generated it.

Reply:
Okay, let's leave it in place then.